examples/ui/grid.rs

```diff
@@ -169,34 +169,31 @@ fn spawn_layout(mut commands: Commands, asset_server: Res<AssetServer>) {
                 ..default()
             });
 
-            // Modal (absolutely positioned on top of content - uncomment to view)
-            // builder.spawn(NodeBundle {
-            //     style: Style {
-            //         position_type: PositionType::Absolute,
-            //         margin: UiRect {
-            //             top: Val::Px(100.),
-            //             bottom: Val::Auto,
-            //             left: Val::Auto,
-            //             right: Val::Auto,
-            //         },
-            //         size: Size {
-            //             width: Val::Percent(60.),
-            //             height: Val::Px(300.),
-            //         },
-            //         max_size: Size {
-            //             width: Val::Px(600.),
-            //             height: Val::Auto,
-            //         },
-            //         ..default()
-            //     },
-            //     background_color: BackgroundColor(Color::Rgba {
-            //         red: 255.0,
-            //         green: 255.0,
-            //         blue: 255.0,
-            //         alpha: 0.8,
-            //     }),
-            //     ..default()
-            // });
+            // Modal (absolutely positioned on top of content - change its visibility to view)
```

Comment:
we can't change the visibility to "view", but "to view", I can see this misinterpreted and users trying to write `Visibility::View`....
```suggestion
            // Modal (absolutely positioned on top of content - currently hidden: to view it, change its visibility)
```